spec/directedGraph_test.js

```diff
@@ -0,0 +1,164 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DirectedGraph from '../src/directedGraph'
+
+chai.use(chaiChange)
+
+describe('DirectedGraph', () => {
+  'use strict'
+
+  let verts = ['a','b','c','d','e','f','g','h','i','j','k','l','m','n','o','p']
+  let paths = [['a','b',3],['a','f',3],['a','e',2],['b','c',1],
+              ['b','h',2],['c','d',2],['c','g',1],['c','l',3],
+              ['d','p',1],['e','f',2],['e','i',1],['f','d',1],
+              ['f','i',2],['f','m',1],['f','k',3],['g','j',1],
+              ['g','p',2],['h','g',1],['h','k',3],['h','l',2],
+              ['i','m',3],['j','i',2],['k','n',1],['k','o',3],
+              ['a','b',2],['m','p',3],['n','o',1],['o','p',2]
+  ]
+
+  it('exists', () => {
+    expect(DirectedGraph).to.be.a('function')
+  })
+
+  context('addVertex() & hasVertex', () => {
```

Comment:
Mocha documentation strongly encourages the use of function() {} instead of the ES6 construct of () => {}. This is just advice passed on from my own reviews done by Jared.